hoot-core/src/main/cpp/hoot/core/io/OsmApiWriter.cpp

```diff
@@ -73,14 +77,17 @@ OsmApiWriter::OsmApiWriter(const QUrl& url, const QList<QString>& changesets)
 bool OsmApiWriter::apply()
 {
   Timer timer;
-  HootNetworkRequestPtr request(new HootNetworkRequest());
+  //  Setup the network request object without OAuth for the capabilities call
+  HootNetworkRequestPtr request = createNetworkRequest();
```

Comment:
Since we create new network request objects all over, this function was created to create the object with or without OAuth depending on the situation.